src/Catel.Core/Catel.Core.Shared/IoC/Interfaces/ITypeRequestPath.cs

```diff
@@ -33,28 +27,5 @@ public interface ITypeRequestPath
         /// </summary>
         /// <value>The last type.</value>
         TypeRequestInfo LastType { get; }
-
-        /// <summary>
-        /// Gets a value indicating whether this path is valid, which means that the same type does not occur multiple times.
-        /// </summary>
-        /// <value><c>true</c> if this instance is valid; otherwise, <c>false</c>.</value>
-        bool IsValid { get; }
```

Comment:
I have added validation inside of factory methods. so there is no possibility to create invalid path